### DISCUSSION TITLE
Syntax of the input files

### DISCUSSION THREAD

**Discussion by EtienneSwerim** [idaholab/moose#16451] [Q&A Modules: General] [answered]
Hi!
Sorry for this (almost embarrassing) beginner question: in the input files, why are sometimes variables (and kernels) simply -like in the workshop- declared as:
[Variables]
   [myVar]
[]
and sometimes as:
[Variables]
   [./myVar]
   [../]
[]
Thanks!
Etienne

**Comment by WilkAndy** [comment, marked as answer]
Hi Etienne, and welcome to MOOSE.   Good question.  Either way is fine.  The first way the modern syntax, while the second way is old-fashioned.

andy

**Comment by EtienneSwerim** [comment]
Hi Andy,
Thank you! OK, so the two ways are completely identical. I thought there was something about under-directories that I had missed, but no. Good to know!
Etienne
